api/v1/controllers/subjects.js

```diff
@@ -415,7 +415,6 @@ module.exports = {
      */
     if (featureToggles.isFeatureEnabled(sampleStoreConstants.featureName) &&
```

Comment:
Looks good.  Just the comment in line 411 needs to be updated to remove the words "fast fail".